src/structures/MessageEmbed.js

```diff
@@ -214,6 +214,17 @@ class MessageEmbed {
       (this.footer ? this.footer.text.length : 0));
   }
 
+  /**
+   * Adds a field to the embed (max 25).
+   * @param {StringResolvable} name The name of this field
+   * @param {StringResolvable} value The value of this field
+   * @param {boolean} [inline] If this field will be displayed inline
```

Comment:
should be `[inline=false]` to clearly indicate that by default a field will not be inline.